SQA_AT/src/main/java/model/Order.java

```diff
@@ -0,0 +1,70 @@
+package model;
+
+public class Order {
```

Comment:
To avoid boilerplate code pls check the lombok library
https://auth0.com/blog/a-complete-guide-to-lombok/